client/buildpacks.go

```diff
@@ -125,16 +126,20 @@ func (b *buildpacksFunctionServer) run() (func(), error) {
 	if err != nil {
 		return nil, err
 	}
-
+	var temp = b.envs.String()
```

Comment:
Since we're printing the full docker command with args below, I think this printout is unnecessary.